docs/explanations/modeling_extensions/surrogate/api/pysmo/pysmo_polyregression.rst

```diff
@@ -102,10 +102,14 @@ The result of the ``pysmo_surrogate.PysmoPolyTrainer`` method is a python object
 
 Confidence intervals for *PysmoPolyTrainer* models
 --------------------------------------------------------------------
-**Coming soon**
 
-The confidence intervals for the regression paramaters may be viewed using the method ``confint_regression``.
+PySMO provides the user with the capability to compute confidence intervals for the regression paramaters using the ``get_confidence_intervals`` method. This can be done by passing the result of the model training and the confidence interval value (*default=0.95*) into the ``pysmo_surrogate.PysmoPolyTrainer`` object:
 
+.. code-block:: python
+  
+  >>> conf_int = pr_trainer.get_confidence_intervals(poly_train, confidence=0.99)
+
+The resulting object ```conf_int`` is a dictionary containing upper and lower confidence bounds as well as the estimated standard errors for all of the regression parameters of the trained models in ``poly_train``. 
```

Comment:
I think you should say what (dictionary) keys are used for these values

Reply:
Done.